core/sphinx/kemsphinx_test.go

```diff
@@ -40,30 +40,57 @@ func TestKEMSphinxSimple(t *testing.T) {
 	t.Parallel()
 	mykem := hybrid.Kyber768X25519()
 	withSURB := false
-	geo := KEMGeometryFromUserForwardPayloadLength(mykem, 512, withSURB, 5)
-	sphinx := NewKEMSphinx(mykem, geo)
+	g := geo.KEMGeometryFromUserForwardPayloadLength(mykem, 512, withSURB, 5)
```

Comment:
here is a case where we aren't using (or testing) `2000` for example?

Reply:
lol yes of course it's works with ANY payload size. And here's another example of that. If you need more examples I can modify more tests.